src/Client.php

```diff
@@ -281,6 +281,11 @@ private function prepareEvent(Event $event, ?EventHint $hint = null, ?Scope $sco
             return null;
         }
 
+        $event = $this->applyIgnoreOptions($event, $hint);
+        if (null === $event) {
```

Comment:
Nitpick, I find this to be more readable and consistent with the rest of the codebase but feel free to ignore

```suggestion
        $event = $this->applyIgnoreOptions($event, $hint);
        
        if (null === $event) {
```